easybuild/easyconfigs/x/X11/X11-20231019-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,265 @@
+    {'libfontenc-1.1.7.tar.gz': '5e5f210329823f08f97bfe9fd5b4105070c789bc5aef88ce01d86d8203d4aa9f'},
+    {'libXfont-1.5.4.tar.gz': '59be6eab53f7b0feb6b7933c11d67d076ae2c0fd8921229c703fc7a4e9a80d6e'},
+    {'libXfont2-2.0.6.tar.gz': 'a944df7b6837c8fa2067f6a5fc25d89b0acc4011cd0bc085106a03557fb502fc'},
+    {'libXft-2.3.8.tar.gz': '32e48fe2d844422e64809e4e99b9d8aed26c1b541a5acf837c5037b8d9f278a8'},
+    {'libXi-1.8.1.tar.gz': '3b5f47c223e4b63d7f7fe758886b8bf665b20a7edb6962c423892fd150e326ea'},
+    {'libXinerama-1.1.5.tar.gz': '2efa855cb42dc620eff3b77700d8655695e09aaa318f791f201fa60afa72b95c'},
+    {'libXrandr-1.5.4.tar.gz': 'c72c94dc3373512ceb67f578952c5d10915b38cc9ebb0fd176a49857b8048e22'},
+    {'libXres-1.2.2.tar.gz': '8abce597ced4a7ab89032aee91f6f784d9960adc772b2b59f17e515cd4127950'},
+    {'libXtst-1.2.4.tar.gz': '01366506aeb033f6dffca5326af85f670746b0cabbfd092aabefb046cf48c445'},
+    {'libXv-1.0.12.tar.gz': 'ce706619a970a580a0e35e9b5c98bdd2af243ac6494c65f44608a89a86100126'},
+    {'libXvMC-1.0.13.tar.gz': 'e630b4373af8c67a7c8f07ebe626a1269a613d262d1f737b57231a06f7c34b4e'},
+    {'libXxf86dga-1.1.6.tar.gz': '87c7482b1e29b4eeb415815641c4f69c00545a8138e1b73ff1f361f7d9c22ac4'},
+    {'libXxf86vm-1.1.5.tar.gz': 'f3f1c29fef8accb0adbd854900c03c6c42f1804f2bc1e4f3ad7b2e1f3b878128'},
+    {'libdmx-1.1.5.tar.gz': '070e82cc1daa1b21ee1339aef56a909eab04cbe7d430fabfbb01ecd21b2dd9f3'},
+    {'libxkbfile-1.1.2.tar.gz': 'd1a7e659bc7ae1aa1fc1ecced261c734df5ad5d86af1ef7a946be0e2d841e51d'},
+    {'libxshmfence-1.3.2.tar.gz': 'e93a85099604beb244ee756dcaf70e18b08701c1ca84c4de0126cd71bd6c8181'},
+    {'xcb-util-0.4.1.tar.gz': '21c6e720162858f15fe686cef833cf96a3e2a79875f84007d76f6d00417f593a'},
+    {'xcb-util-image-0.4.1.tar.gz': '0ebd4cf809043fdeb4f980d58cdcf2b527035018924f8c14da76d1c81001293b'},
+    {'xcb-util-keysyms-0.4.1.tar.gz': '1fa21c0cea3060caee7612b6577c1730da470b88cbdf846fa4e3e0ff78948e54'},
+    {'xcb-util-renderutil-0.3.10.tar.gz': 'e04143c48e1644c5e074243fa293d88f99005b3c50d1d54358954404e635128a'},
+    {'xcb-util-wm-0.4.2.tar.gz': 'dcecaaa535802fd57c84cceeff50c64efe7f2326bf752e16d2b77945649c8cd7'},
+    {'xcb-util-cursor-0.1.5.tar.gz': '0e9c5446dc6f3beb8af6ebfcc9e27bcc6da6fe2860f7fc07b99144dfa568e93b'},
+    {'xkeyboard-config-2.40.tar.xz': '7a3dba1bec7dc7191432da021242d17c9cf6c89690e6c57b0de048ff8c9d2ae3'},
+    {'printproto-1.0.5.tar.gz': 'e8b6f405fd865f0ea7a3a2908dfbf06622f57f2f91359ec65d13b955e49843fc'},
+    {'libXp-1.0.4.tar.gz': '05e46af1ccb68f1752cca5879774a4fb9bf3b19fe088eb745034956e0c6fadba'},
+    {'xbitmaps-1.1.3.tar.gz': '93b433b7ff223c4685fdba583b4bd30f2706be2413a670021084422d85b0269d'},
+]
+
+default_easyblock = 'ConfigureMake'
+
+default_component_specs = {
+    'sources': [SOURCE_TAR_GZ],
+    'start_dir': '%(name)s-%(version)s',
+}
+
+components = [
+    ('libpthread-stubs', '0.5', {  # 2023-07-18
+        'checksums': ['50d5686b79019ccea08bcbd7b02fe5a40634abcfd4146b6e75c6420cc170e9d9'],
```

Comment:
This checksum is wrong, should be `593196cc746173d1e25cb54a93a87fd749952df68699aab7e02c085530e87747` (as in `checksums` above).

We should remove the top-level `checksums`, and make sure the per-component checksum is correct (however annoying that is, since `--inject-checksums` on components doesn't work currently)